03/app.js

```diff
@@ -1,17 +1,70 @@
 const min = 1;
 const max = 100;
 
-const a = randomNumber(min, max);
-const b = randomNumber(min, max);
-const c = randomNumber(min, max);
+// const a = randomNumber(min, max);
+// const b = randomNumber(min, max);
+// const c = randomNumber(min, max);
 
-console.log(a, b, c);
+// console.log(a, b, c);
 
+const isEven = checkEven;//nie czaję, czy to tak powinno być. Kiedy od razu przypisywałam funkcję anonimową,
```

Comment:
dlaczego od razu nie stworzyłaś funkcji `isEven`?

Reply:
XD to już pozostanie zagadką